airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -137,6 +138,10 @@ public class ConnectionManagerWorkflowImpl implements ConnectionManagerWorkflow
   @Trace(operationName = WORKFLOW_TRACE_OPERATION_NAME)
   @Override
   public void run(final ConnectionUpdaterInput connectionUpdaterInput) throws RetryableException {
+    if (connectionId == null) {
+      connectionId = Objects.requireNonNull(connectionUpdaterInput.getConnectionId());
+      log.warn("connectionId is null for workflow {}; setting to input connection {}", workflowState.getId(), connectionId);
+    }
```

Comment:
@justenwalker I think to validate the change we must have a unit test to ensure this is solving the problem.

Reply:
Can we remove the warning? This will happens for all the connection. This is similar to what we are doing [here](https://github.com/airbytehq/airbyte-platform-internal/blob/595822e05cfd8383601028985af6f741ea9ec25b/oss/airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java#L190). I think

Reply:
I'd like to; but its not trivial. There is a ton of indirection and class loading stuff that happens in the temporal layer that is difficult to get visibility into. Perhaps someone more experienced with this can give me some guidance?

Reply:
@benmoriceau I've removed the message. Still unsure how to test this though given the amount of magic temporal stuff going on.